run.sh

```diff
@@ -11,7 +11,13 @@ node node_modules/gulp/bin/gulp build-sass
 # If LOCALSTACK is set in environment, this will upload static files to the localstack s3 service running in docker
 # Otherwise the development server is handling static files
 if [[ -n "${USE_LOCALSTACK}" ]]; then
+    echo Removing crt-portal s3 bucket
```

Comment:
We've been seeing some inconsistent behavior with respect to the static files updating/not updating in the localstack s3 bucket when running locally, which can frustrate development and debugging efforts. This is fairly blunt, but effective at ensuring that you have the latest stuff in your static bucket for local development and testing